.github/workflows/docs.yml

```
@@ -102,7 +102,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5.6.0
         with:
-          python-version: '3.10'
+          python-version: '3.11'
           
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)